web/modules/custom/collation_fixer/README

```diff
@@ -0,0 +1,41 @@
+# Collation Fixer
+
+A 'runtime' requirements checker (admin/reports/status) for checking
+all database tables have the right database collation (probably very
+MySQL centric).
+
+Offers the ability to fix tables with wrong database collation.
+
+The collation is either specified in your database settings.
+
+This can be useful when developing non-english sites.
+
+Example of specifying in database settings in settings.php (will
+affect all tables - it's probably better to do it in hook_schema() or
+hook_schema_alter()):
```

Comment:
```suggestion
Example of specifying in database settings in settings.php (will
affect all tables):
```
Røg `hook_schema()`-understøttelse ikke ud?

Reply:
I have updated the README.